cmd/external-repos/main.go

```diff
@@ -45,9 +45,11 @@ func main() {
 		if len(args) < 3 {
 			log.Fatal().Msg("Usage:  ./external_repos introspect URL")
 		}
-		count, err := external_repos.IntrospectUrl(args[2])
+		count, errors := external_repos.IntrospectUrl(args[2])
 		if err != nil {
-			log.Panic().Err(err).Msg("Failed to introspect repositories")
+			for i := 0; i < len(errors); i++ {
+				log.Panic().Err(errors[i]).Msg("Failed to introspect repository")
+			}
```

Comment:
Unwrap this block from the `if`block and remove the `if err != nil` statement; it is checking a different variable and won't be needed as if no error `len(error)` will be 0 and loop iteration is excuted.